peerinst/templates/peerinst/collection/collection_detail.html

```diff
@@ -0,0 +1,324 @@
+{% extends 'peerinst/base.html' %}
+
+{% load get_value bleach_html staticfiles compress i18n %}
+
+{% block stylesheets %}
+{{ block.super }}
+{% compress css %}
+<link href="{% static 'peerinst/css/collection.min.css' %}" rel="stylesheet">
+{% endcompress %}
+{% endblock %}
+{% block body %}
+<main>
+  <section>
+    <h1 class="mdc-typography--display3">{% trans 'Collections' %}
+      <svg class="underline" width=150 height=4></svg>
+    </h1>
+    <div class="admin-link">
+      <a href="{% url 'teacher' request.user.teacher.pk %}">{% trans 'Back to My Account' %}</a>
+      <span class="links-divider">|</span><a href="{% url 'collection-list' %}">{% trans 'View Collections' %}</a>
+      <span class="links-divider">|</span>{% if request.user.teacher == object.owner %}<a href="{% url 'collection-update' object.pk %}">{% trans 'Edit' %}</a>
```

Comment:
Split lines so they're below 80 characters.

Reply:
Be careful though: line breaks in the html will change the horizontal spacing.